src/nodes/agent/handlers/VaultsGitPackGet.ts

```diff
@@ -103,7 +102,7 @@ class VaultsGitPackGet extends RawHandler<{
         sideBand.destroy(e);
       },
     });
-    return [null, outputStream];
+    return [{}, outputStream];
```

Comment:
So now if it is an empty object, I'm assuming that this is not the `result` key.

![image](https://github.com/MatrixAI/Polykey/assets/640797/1d70eae6-2eff-4c15-9ca7-54070c9a2000)

Otherwise what was the reason to switch from `null` to `{}`?

Reply:
it is the result key, because the result MUST be a JSON object now. So that the metadata can fit in there